scripts/serverless.js

```diff
@@ -647,7 +650,9 @@ const processSpanPromise = (async () => {
       }
 
       if (!isTelemetryDisabled && !isHelpRequest) {
-        await storeTelemetryLocally(await generateTelemetryPayload(serverless));
+        const telemetryPayload = await generateTelemetryPayload(serverless);
+        await storeTelemetryLocally({ ...telemetryPayload, outcome: 'success' });
+        hasTelemetryBeenReported = true;
```

Comment:
I'd put it at the begin of this block, as I wouldn't reattempt to store telemetry if any of telemetry utils crashes